MailCore/API/Endpoint.swift

```diff
@@ -190,6 +190,10 @@ public extension Endpoint {
         ])
     }
 
+    static func externalMailFlag(hostingId: Int, mailboxName: String) -> Endpoint {
+        return .baseManager.appending(path: "/\(hostingId)/mailboxes/\(mailboxName)/external_mail_flag")
```

Comment:
We now have multiple endpoints starting with `.baseManager.appending(path: "/\(hostingId)/mailboxes/\(mailboxName)/`, would it be possible to refactor?